src/status_im/constants.cljs

```diff
@@ -26,3 +27,28 @@
 
 (def console-chat-id "console")
 (def wallet-chat-id "wallet")
+
+(def default-network "testnet_rpc")
+(def default-networks
+  {"mainnet"     {:id     "mainnet",
```

Comment:
do we really want to give access to mainnet in this release? @sla-shi ?

Reply:
mainnet should be disabled until we tested switching between testnets @flexsurfer

Reply:
so we need to implement this in UI then @rasom , disable mainnet in the list